File: go/token_response.go
// this file is @generated
package meteroid

// Token response as per OAuth 2.0 spec
type TokenResponse struct {
	AccessToken string `json:"access_token"`

	ExpiresIn int64 `json:"expires_in"`

	RefreshToken *string `json:"refresh_token,omitempty"`

	Scope *string `json:"scope,omitempty"`

	TokenType string `json:"token_type"`
}

File: python/meteroid/models/token_response.py
# this file is @generated
import dataclasses
import typing as t

from ..serialization import BaseModel


@dataclasses.dataclass
class TokenResponse(BaseModel):
    """Token response as per OAuth 2.0 spec"""

    access_token: str

    expires_in: int

    token_type: str

    refresh_token: t.Optional[str] = None

    scope: t.Optional[str] = None
